src/orion/core/io/cmdline_parser.py

```diff
@@ -91,9 +91,26 @@ def format(self, configuration):
         # Arguments with spaces are broken if we use ' '.join(template).format().split(' ').
         # Hence we iterate over the list as-is and format on each item separately.
         formatted = []
+        item: str
```

Comment:
lost typing :)

Reply:
Not really, this is intentional: this makes the code below aware that the `item` variable is of type `str`.

Reply:
I meant that it is a type annotation in a non-typed module.

Reply:
My interpretation of your "type/no-type" directive was that it was about the public-facing API (methods signatures, etc).
There is really no harm in leaving this in, is there?

Reply:
I see. I have a different opinion on this: I think incremental addition of typing is fine, and I'd say that adding typing inside methods or internal functions is a useful first step towards modernizing the codebase. I'll remove this for now, just so you know, my opinion is that we should embrace these small quality-of-life changes, rather than dis-incentivize them.

Reply:
Fixed in 8541015d207f0293ee5894f53bf8fbe7e52e9539